src/test/java/app/studnicki/ox/ScoreTest.java

```diff
@@ -0,0 +1,68 @@
+package app.studnicki.ox;
+
+//STATIC
+
+import static org.testng.Assert.assertEquals;
+
+//NON STATIC
+
+import app.studnicki.ox.game.Score;
+import org.testng.annotations.Test;
+
+
+@Test
+public class ScoreTest {
```

Comment:
Można by popracować nad nazwami metod testowych np. *add* zamiast *added* albo tej ostatniej tak by  podpadała pod konwencję `should...`. Zasadniczo obierz 1 konwencję i się jej trzymaj.

Tu stosowny artykuł: https://dzone.com/articles/7-popular-unit-test-naming